django/http/multipartparser.py

```diff
@@ -656,11 +656,12 @@ def parse_boundary_stream(stream, max_header_size):
     header_end = chunk.find(b"\r\n\r\n")
 
     def _parse_header(line):
-        main_value_pair, params = parse_header(line)
+        main_value_pair, params = parse_header_parameters(line.decode())
+        params = {k: v.encode() for k, v in params.items()}
         try:
             name, value = main_value_pair.split(":", 1)
         except ValueError:
-            raise ValueError("Invalid header: %r" % line)
+            raise ValueError("Invalid header: %r" % line.decode())
```

Comment:
I removed decode() from that line. But, in the loop in multipartparser.parse_boundary_stream - the only where that we call _parse_header - we skip the exception. Should I make the function throw an exception to create a test case for the message?

Reply:
True, this message is unused since its introduction in d725cc9734272f867d41f7236235c28b3931a1b2. I think we can remove `_parse_header()` hook, see #15802.